travis-autobuild-extensions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ "$TRAVIS_REPO_SLUG" == "new-xkit/XKit" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+  echo -e "Rebuilding extension dist...\n" &&\
+
+  cd $HOME &&\
+  git config --global user.email "travis@travis-ci.org" &&\
+  git config --global user.name "travis-ci" &&\
+  git clone https://"$GH_TOKEN"@github.com/new-xkit/XKit &&\
```

Comment:
The GH Token is an environment variable I already set for our travis builds. The token itself comes from the NewXKitBot